Content.Client/Chat/Managers/ContentMarkupTagManager.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Linq;
+using Content.Client.Chat.MarkupTags;
+using Content.Shared.Chat;
+using Content.Shared.Chat.ContentMarkupTags;
+
+namespace Content.Client.Chat.Managers;
+
+public sealed class ContentMarkupTagManager : ISharedContentMarkupTagManager
+{
+    public Dictionary<string, IContentMarkupTag> ContentMarkupTagTypes => new IContentMarkupTag[]
```

Comment:
u decalare it as Dictionary, so u are OK with changing it in runtime? I'm not 100% sure what about maintainers policy on that but i'd better declare it as IReadOnlyDictionary if u want it initialized and never changed after creation. Maybe some frozen dictionary for bettere perf but same semantics?